frontend/test/unit/updates/mntner.service.spec.ts

```diff
@@ -406,22 +403,50 @@ describe('MntnerService', () => {
         expect(mntnerService.removeDuplicateMntsFromAttribute(attributes).length).toBe(5);
     });
 
-    it('should not stripNccMntners for other env except PROD env', () => {
-        mntnerService.properties.NO_PASSWORD_AUTH_POPUP = true;
+    it('should strip already listed autocomplete mntners', () => {
```

Comment:
this should be renamed to filterAutocompleteMntners, because is testing filterAutocompleteMntners not stripNccMntners. Same apply for next 2 tests